src/cbc_sdk/enterprise_edr/ubs.py

```diff
@@ -36,6 +36,13 @@ class Summary(UnrefreshableModel):
         urlobject_single = "/ubs/v1/orgs/{}/sha256/{}/summary/device"
 
         def __init__(self, cb, model_unique_id):
+            """
+            Initialize the Summary object.
```

Comment:
I think this should be "Initialize the Binary.Summary object."

Reply:
Or keep it as Summary, depending on your design choice as mentioned in a comment in `enterprise_edr.base`

Reply:
Keeping it as Summary in light of the above.